superset-frontend/src/components/Select/SupersetStyledSelect.tsx

```diff
@@ -243,6 +244,18 @@ function styled<
       });
     }
 
+    // handle forcing dropdown overflow
+    // use only when setting overflow:visible isn't possible on the container element
+    if (forceOverflow) {
```

Comment:
Are there any places that have this option already as `true`?

Reply:
Yes you can see two instances in this PR already. One is on the dashboard filters and the other one is used for the refresh-interval modal. The one for the dashboard filters is required. The one for refresh-interval is more to show how it looks like on a modal but it is not necessary and can be removed.

Reply:
I see, thanks for explanation 👍